File: app/src/main/java/dstudio/com/retrofitrxandroid/apiconnection/APIInterface.java
package dstudio.com.retrofitrxandroid.apiconnection;


import dstudio.com.retrofitrxandroid.model.userdataresponse.UserData;
import retrofit2.http.GET;
import rx.Observable;

/**
 * Created by janwelcris on 6/8/2017.
 */

public interface APIInterface {
    @GET("/janwelocampo/AndroidTutorial/master/collection.json")
    Observable<UserData> doGetUserData();



}

File: app/src/main/java/dstudio/com/rxkotlinandroid/retrofiturl/APIInterface.kt
package dstudio.com.rxkotlinandroid.retrofiturl

import dstudio.com.rxkotlinandroid.model.userdataresponse.UserData
import retrofit2.http.GET
import rx.Observable

/**
 * Created by janwelcris on 7/12/2017.
 */

interface APIInterface {

    @GET("/amaysim-au/engineering-test-resources/master/collection.json")
    fun doGetUserData(): Observable<UserData>
}
